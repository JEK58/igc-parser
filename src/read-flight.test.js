const fs = require('fs');
const readFlight = require('./read-flight');

describe('readFlight()', () => {
  it(`reads "SW_77flqgg1" flight log correctly`, () => {
    let flight = readFlight(`${__dirname}/../fixtures/2017-07-15-lev/SW_77flqgg1.IGC`);
    expect(flight).toMatchSnapshot();
  })
});
