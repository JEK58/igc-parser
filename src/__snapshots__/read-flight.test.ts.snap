// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`readFlight() reads "SW_77flqgg1" flight log correctly 1`] = `
Object {
  "altitude": 44,
  "coordinate": Array [
    7.01,
    51.01108333333333,
  ],
  "time": 1500111903000,
  "valid": true,
}
`;

exports[`readFlight() reads "SW_77flqgg1" flight log correctly 2`] = `
Object {
  "altitude": 1168,
  "coordinate": Array [
    6.52835,
    50.86615,
  ],
  "time": 1500118283000,
  "valid": true,
}
`;

exports[`readFlight() reads "SW_77flqgg1" flight log correctly 3`] = `
Object {
  "altitude": 671,
  "coordinate": Array [
    7.0655833333333335,
    51.09908333333333,
  ],
  "time": 1500128791000,
  "valid": true,
}
`;
